src/components/Home/HomepageMap.tsx

```diff
@@ -67,33 +60,21 @@ export default function HomepageMap() {
         });
     };
 
-    useMemo(() => {
```

Comment:
Same question as above.

Reply:
please find the  reply in the comment here https://github.com/Weath-r/fwm-app/pull/31#discussion_r1489794408